src/stores/customGroups.ts

```
@@ -0,0 +1,515 @@
+    >();
+
+    groups.forEach((group) => {
+      if (!availability.has(group.name)) {
+        availability.set(group.name, {
+          locales: new Set(),
+          gameModes: new Set(),
+          combinations: [],
+        });
+      }
+
+      const groupData = availability.get(group.name)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)